src/tests.rs

```diff
@@ -59,6 +60,22 @@ fn parse_file() {
 }
 
 #[test]
+fn read_write_file() {
+    let mut path = Path::new(env!("CARGO_MANIFEST_DIR")).to_path_buf();
```

Comment:
Please don’t have tests pollute the source directory. Use https://crates.io/crates/tempdir as a dev-dependency instead.